packages/notifications/resources/views/database-notifications.blade.php

```diff
@@ -1,6 +1,7 @@
 @php
     $notifications = $this->getNotifications();
     $unreadNotificationsCount = $this->getUnreadNotificationsCount();
+    $markAllNotificationsAsReadAction = $this->markAllNotificationsAsReadAction();
```

Comment:
Since you're now using `InteractsWithActions`, you can access the action as a property directly, and it will be cached

```php
:mark-all-notifications-as-read-action="$this->markAllNotificationsAsReadAction"
```